CHANGELOG.md

```diff
@@ -4,6 +4,17 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 
+v1.6.0
```

Comment:
Could you bump the version number (and date)? Just merged another PR which ended up as v1.6.0.